sw/device/tests/sim_dv/pwrmgr_random_sleep_power_glitch_reset_test.c

```diff
@@ -466,20 +466,24 @@ bool test_main(void) {
     case 0:
       if (RST_IDX[event_idx] % 2) {
         LOG_INFO("Booting and setting normal sleep followed by glitch reset");
+        LOG_INFO("Let SV wait timer reset");
```

Comment:
nit: do we need this print, the earlier one seems providing enough info?

Reply:
When vseq file has no activity, the loop can proceed repeatedly from the existing message. line 469 is to avoid multiple iteration from one message.